crates/router/src/connector/payu/transformers.rs

```diff
@@ -22,6 +22,7 @@ pub struct PayuPaymentsRequest {
     description: String,
     pay_methods: PayuPaymentMethod,
     continue_url: Option<String>,
+    ext_order_id: String,
```

Comment:
Make this field optional as suggested in payU docs and also update the branch to latest main to avoid spell check failure.